### PR TITLE
Component ScrollTable

### PR DESCRIPTION
**Pull request by ogupte**
## PR Checklist

- Manually tested across supported browsers
  - [x] Chrome
  - [x] Firefox
  - [x] Safari
  - [x] IE11 (Win 7)
  - [x] Edge (Win 10)
- [x] Unit tests written (`common` at minimum)
- [x] PR has one of the `semver-` labels
- [x] Two core team engineer approvals
- [x] One core team UX approval

### CODE REVIEW

File: src/components/ScrollTable/ScrollTable.jsx [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+	},
+
+	render() {
+		const {
+			children,
+			className,
+			style,
+			tableWidth,
+			isNowrap,
+			...passThrus
```
Comment (sodiumjoe) [inline src/components/ScrollTable/ScrollTable.jsx]: this is spelled `passThroughs` everywhere else

File: src/components/Table/Table.jsx [hunk 2]
```diff
@@ -458,10 +459,16 @@ const Table = React.createClass(createLucidComponentDefinition({
 		 * Adjusts the style of the table to have more spacing within the table cells
 		 */
 		hasExtraWhitespace: bool,
+
+		/**
+		 * render the table without borders on the outer edge
+		 */
+		hasNoBorder: bool,
```
Comment (sodiumjoe) [inline src/components/Table/Table.jsx]: is there a reason for this to be `hasNoBorder` with default `false` instead of `hasBorder` with default `true`?

Comment (ogupte) [inline src/components/Table/Table.jsx]: yes, the css class name is `has-no-border` and it actually removes existing default border styles, so the negation follows suit with that.

File: src/components/ScrollTable/ScrollTable.jsx [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+				className={boundClassNames('&', className)}
+				style={style}
+			>
+				<Table
+					style={{
+						width: tableWidth,
+					}}
+					className={boundClassNames({'&-nowrap': isNowrap})}
+					hasNoBorder={true}
+					{...passThroughs}
```
Comment (jondlm) [inline src/components/ScrollTable/ScrollTable.jsx]: Probably want this above the other props

File: src/components/ScrollTable/ScrollTable.less [hunk 4]
```diff
@@ -0,0 +1,15 @@
+.lucid-ScrollTable {
+	border: @Table-cell-border;
```
Comment (jondlm) [inline src/components/ScrollTable/ScrollTable.less]: Any time we use variables from other components, we need to hoist them up in to `variables.less`

File: src/components/Table/Table.jsx [hunk 5]
```diff
@@ -230,9 +230,6 @@ const Th = React.createClass(createLucidComponentDefinition({
 			passiveWidth: width || null
 		};
 	},
-	componentDidMount() {
-		this._root = this.refs['root'];
```
Comment (jondlm) [inline src/components/Table/Table.jsx]: This did seem redundant to me, so 👍

**Pull request merged**